docs/user-guide/sdh/tenants.md

```diff
@@ -0,0 +1,78 @@
+# Tenants
+
+This section allows you to add new [tenants](#tenants) and map environments to these [installations](./README.md#installations) to ensure updates [rollout](#rollout) correctly. 
+
+## Adding Tenant
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add tenants.
+{% endhint %}
+
+This involves the creation of new organizations where you wish to deploy s/w updates. Whenever you are onboarding a new client, you add them as a tenant. 
+
+1. Click **+ Add Tenant**.
+
+    ![Figure 1: Adding a Tenant](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/tenants-tab.jpg)
+
+2. Enter a name in **Tenant display name** field, e.g., `flareup.xyz`
+
+3. Add a unique identifier to your tenant in **Tenant ID** field, e.g., `flareup123` 
```

Comment:
This cannot be changed

Reply:
Can only use alpha-numeric and `-`

Reply:
1. Agreed on the first suggestion...
2. No, let's not include field validations in the doc especially if they are already shown to the user as they type
<img width="1378" alt="image" src="https://github.com/user-attachments/assets/af2da662-8be8-49fe-9f7c-fde78980000c">